_data/versions.yml

```diff
@@ -1,7 +1,192 @@
+      and `ETCD_SCHEME` environment variables. Calico no longer reads these values. If you have 
+      not transitioned to `ETCD_ENDPOINTS`, you must do so as of v3.0. Refer to 
+      [Configuring `calicoctl` - etcdv3 datastore](https://docs.projectcalico.org/v3.0/reference/calicoctl/setup/etcdv3) for more information.
+
+    - A new node controller for Kubernetes deployments clears data associated with deleted 
+      nodes from the Calico datastore, preventing conflicts that can lead to crash loops. Refer 
+      to [Configuring the Calico Kubernetes controllers](https://docs.projectcalico.org/v3.0/reference/kube-controllers/configuration) for more information.
+
+    ##### Other changes
+    
+    - Calico now works with Kubernetes network services proxy with IPVS/LVS. Calico enforces 
+      network policies with kube-proxy running in IPVS mode for Kubernetes clusters. Currently 
+      only workload ingress policy is supported.
+
+    - Rolling update is now enabled by default in the Kubernetes self-hosted manifests. 
+      [calico #1506](https://github.com/projectcalico/calico/pull/1506) (@caseydavenport)
+      
+    - The CoreOS version used for the Kubernetes Vagrant tutorial has been updated, resolving
+      an issue causing `kubectl` to hang. [calico #1487](https://github.com/projectcalico/calico/pull/1487) (@2ffs2nns)
+      
+    - Typha no longer sends incorrect `updateTypes` to Felix, increasing the accuracy of 
+      Felix statistics. [typha #70](https://github.com/projectcalico/typha/pull/70) (@fasaxc)
+
+    #### Limitations
+
+    - **Offers only Kubernetes, OpenShift, and host endpoint integrations**: the 
+      OpenStack, Mesos, DC/OS, rkt, and Docker orchestrators have not been tested 
+      and are not supported. (Calico still supports Docker and rkt containers.) 
+      We plan to resume support for the OpenStack, Mesos, DC/OS, and Docker 
+      orchestrators in a future release.
+
+    - **GoBGP not supported**: Setting the `CALICO_NETWORKING_BACKEND` environment 
+      variable to `gobgp` is not supported. See [Configuring calico/node](https://docs.projectcalico.org/v3.0/reference/node/configuration) 
+      for more information. We plan to resume support for GoBPG in a future release.
+    
+    - **Route reflectors cannot be clustered**: We plan to resume support for
+      this in a future release.
+
+  components:
+    felix:
+      version: 3.0.1
+      url: https://github.com/projectcalico/felix/releases/tag/3.0.1
+    typha:
+      version: v0.6.0
+      url: https://github.com/projectcalico/typha/releases/tag/v0.6.0
+    calicoctl:
+      version: v2.0.0
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v2.0.0
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v2.0.0/calicoctl
+    calico/node:
+      version: v3.0.1
+      url: https://github.com/projectcalico/calico/releases/tag/v3.0.0
```

Comment:
should be v3.0.1 (I think that's my mistake)

Reply:
OK